dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/MergeableClusterInvoker.java

```diff
@@ -54,7 +54,6 @@
 public class MergeableClusterInvoker<T> extends AbstractClusterInvoker<T> {
```

Comment:
Could you please also check if we can remove some unused import related this change?
for example

```
import java.util.concurrent.ExecutorService;
import java.util.concurrent.Executors;
```

Reply:
done